subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -7998,7 +7998,8 @@ static void event_slave(u32_t ticks_at_expire, u32_t remainder, u16_t lazy,
 #endif /* CONFIG_BT_CTLR_GPIO_LNA_PIN */
 
 #if defined(CONFIG_BT_CTLR_PROFILE_ISR) || \
-    defined(CONFIG_BT_CTLR_GPIO_PA_PIN)
+    defined(CONFIG_BT_CTLR_GPIO_PA_PIN) || \
+    defined(CONFIG_BT_CTLR_SW_SWITCH_SINGLE_TIMER)
```

Comment:
this was necessary, since peripheral role does not capture the END event timestamp on the first packet, otherwise.